src/astro/files/operators/files.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any
+
+from airflow.models import BaseOperator
+from airflow.utils.context import Context
+
+from astro.files.locations import create_file_location
+
+
+class _ListFileOperator(BaseOperator):
```

Comment:
Since we're not prefixing the other operators, it may be worth to not have it now - and to log a micro ticket for us to prefix all of them with _. This would make the code consistent.